<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCard</class>
 <widget class="QWidget" name="FormCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>295</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Colores primarios*/
/* 
gris oscuro #222222 #333333 #444444
gris claro #999999 #DDDDDD
verde claro #C8CC8E
vinotinto #742427
*/
/*Colores secundarios */
/* 
Azules #36C9C6 #00BDB9 #77ACA2
rojos #910D3F #C70039 #F94646
naranjas #D34E24 #F28123 #F7F052
*/

/* Funetes/*

/*
italic: font: italic 9pt &quot;Ubuntu&quot;;
regular: font: 9pt &quot;Ubuntu&quot;;
light: font: 300 9pt &quot;Ubuntu&quot;;
medium: font: 500 9pt &quot;Ubuntu&quot;;
bold: font: 700 9pt &quot;Ubuntu&quot;;
*/

/****************************************************************************************************************************************************/


QLabel#label_cardName{
font: 700 11pt &quot;Ubuntu&quot;;
color: #DDDDDD;
}

QLabel#label_cardDataTime, QLabel#label_cardPath{
font: 500 9pt &quot;Ubuntu&quot;;
color: #999999;
}

QFrame#frame_card{
background: #333333;
border-radius: 5px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_card">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_img">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/iconos_logo/iconos/iconos_logo/Logo_V1.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="label_cardName">
           <property name="text">
            <string>3 SIG 83 obras</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_cardDataTime">
           <property name="text">
            <string>25/02/2022 07:32:52 A.M.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_cardPath">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>D:/Document/Ejem</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
